Makefile

```diff
@@ -18,7 +18,12 @@ VERSION := $(shell cat ./VERSION)
 ifneq ($(GO111MODULE),off)
 	MOD_VENDOR := "-mod=vendor"
 endif
-GO_BUILD := $(GO) build $(MOD_VENDOR) -buildmode=pie $(EXTRA_FLAGS) -tags "$(BUILDTAGS)" \
+ifeq ($(GOOS),linux)
+	ifneq ($(GOARCH),$(filter $(GOARCH),mips mipsle mips64 mips64le ppc64))
```

Comment:
This isn't actually quite right because `GOOS` and `GOARCH` aren't necessarily set, so we need to query them.  I'll update shortly.